shared/actions/chat2/index.tsx

```diff
@@ -3606,6 +3623,7 @@ function* chat2Saga() {
   yield* Saga.chainAction(Chat2Gen.messageEdit, clearMessageSetEditing)
   yield* Saga.chainAction2(Chat2Gen.messageDelete, messageDelete)
   yield* Saga.chainAction2(Chat2Gen.messageDeleteHistory, deleteMessageHistory)
+  yield* Saga.chainGenerator(Chat2Gen.dismissJourneycard, dismissJourneycard)
```

Comment:
this doesn't need to be a generator, please make it a `chainAction`

Reply:
It puts messagesWereDeleted and then calls the RPC. Not sure how to do that with chainAction. action-then-call is slightly better than call-then-action here since the action immediately removes the message from the screen, whereas the rpc could take more time.